tensorflow_hub/resolver.py

```diff
@@ -380,9 +380,12 @@ def atomic_download(handle,
                                              overwrite=False)
         # Must test condition again, since another process could have created
         # the module and deleted the old lock file since last test.
-        if tf_v1.gfile.Exists(module_dir):
+        if tf_v1.gfile.Exists(module_dir) and \
```

Comment:
Line continuations with \ are not allowed in Google's py style-guide, lets do

   if (tf_v1.gfile.Exists(module_dir) and 
       tf_v1.gfile.ListDirectory(module_dir)):
     ...